Makefile

```diff
@@ -11,7 +11,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION  := 0.1.0
-TARGET   := prom2json
+GO    := GO15VENDOREXPERIMENT=1 go
+PROMU := $(GOPATH)/bin/promu
```

Comment:
Ahh, I copied from mysqld_exporter, I guess we need to have a little script that checks to see if these templates are all up to date on each repo.